packages/tldraw/src/lib/ui/hooks/useActions.tsx

```diff
@@ -184,9 +185,11 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 			},
 			{
 				id: 'export-as-json',
-				label: 'action.export-as-json',
-				menuLabel: 'action.export-as-json.short',
-				contextMenuLabel: 'action.export-as-json.short',
+				label: {
+					default: 'action.export-as-json',
+					menu: 'action.export-as-json.short',
+					['context-menu']: 'action.export-as-json.short',
```

Comment:
reading all of these labels makes me think i wish it was:
```
  default: 'action.export-as-json',
  short: 'action.export-as-json.short',
```

and `short` is used where appropriate, if present. thoughts?

Reply:
I think we tried something like this at one point, but it seemed simpler to make it specific to specific UI components. It avoids unclear things like `short`, `shorter`, etc